engine/pom.xml

```diff
@@ -145,7 +145,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <configuration>
-          <maxAllowedViolations>2320</maxAllowedViolations>
+          <maxAllowedViolations>2063</maxAllowedViolations>
```

Comment:
Is the number reduced due to change to the apex_checks.xml or code style violations were fixed as part of other commits?